pkg/views/login/create.go

```diff
@@ -61,5 +62,12 @@ func CreateView(loginView *LoginView) {
 	if err != nil {
 		log.Fatal(err)
 	}
+	TrimSpaceOfLoginInputs(loginView)
+}
 
+func TrimSpaceOfLoginInputs(loginView *LoginView) {
```

Comment:
Transfer this logic to a helper function and invoke it directly within the `login.go` file

Reply:
Done.